client/src/generated/graphql.tsx

```diff
@@ -11,15 +11,19 @@ export type Scalars = {
   Boolean: boolean;
   Int: number;
   Float: number;
+  /** Use JavaScript Date object for date/time fields. */
+  DateTime: any;
```

Comment:
instead of any we could type it by adding the following property in the codegen file
```
    config:
      scalars:
        DateTime: string
```